### DISCUSSION TITLE
Rapport de fin de fight : Messages Spéciaux quand non-réaction

### DISCUSSION THREAD

**Discussion by leuhvraiminoryo** [Crownicles/Crownicles#1963] [Idées acceptées / déjà implémentées]
Il serait bien d'avoir des textes spéciaux dans l'message du rapport de gloire en fin de fight, lorsque ledit fight est terminé à cause de l'inaction d'un des joueurs.

ex : 

![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/50586611/fcdb0549-ce2b-414d-9c24-cb9632980e5b)

j'viens d'avoir ça alors que mon adversaire était pas là et a pas fait la moindre action, ça passe moyen :slightly_frowning_face:

**Comment by BastLast** [comment]
Approuvé
